django_saml2_auth/views.py

```diff
@@ -105,6 +105,10 @@ def acs(request: HttpRequest):
     # login via sp_initiated_login endpoint, or it could be a URL used for redirection.
     relay_state = request.POST.get("RelayState")
     relay_state_is_token = is_jwt_well_formed(relay_state) if relay_state else False
+    if next_url is None and relay_state:
```

Comment:
I think you should also check if the `relay_state` is not a token, because you might end up seeing a token instead of a URL in the `next_url` field:
```suggestion
    if next_url is None and relay_state and not relay_state_is_token:
```
WDYT?

Reply:
Thanks, yeah makes sense I have added additional check to see its not a token